recipe/build.sh

```diff
@@ -204,15 +238,16 @@ case ${PKG_NAME} in
 
     mv build/lib.*/torch/bin/* ${PREFIX}/bin/
     mv build/lib.*/torch/lib/* ${PREFIX}/lib/
-    mv build/lib.*/torch/share/* ${PREFIX}/share/
+    # need to merge these now because we're using system pybind11, meaning the destination directory is not empty
+    rsync -a build/lib.*/torch/share/* ${PREFIX}/share/
```

Comment:
I would feel safer having an explicit `build` dependency on rsync. This doesn't look a tool one could reliably expect to be present on host.

Reply:
done